mod/groups/views/default/resources/groups/owner.php

```diff
@@ -8,8 +8,7 @@
 	$title = elgg_echo('groups:owned:user', [$page_owner->getDisplayName()]);
 }
 
-elgg_push_breadcrumb(elgg_echo('groups'), "groups/all");
-elgg_push_breadcrumb($title);
+elgg_push_breadcrumb(elgg_echo('groups'), 'groups/all');
```

Comment:
if you're going to make things nicer, also use a route

Reply:
you could spend a day fixing all hard coded urls at the moment... not considering this a blocking issue